controllers/tasks.js

```diff
@@ -52,8 +52,15 @@ const fetchTasks = async (req, res) => {
  */
 const getUserTasks = async (req, res) => {
   try {
+    const status = req.query.status
     const { username } = req.params
-    const allTasks = await tasks.fetchUserTasks(username)
+    let allTasks = []
+
+    if (status) {
+      allTasks = await tasks.fetchUserTasks(username, status)
```

Comment:
status needs to go in as an array
`[status]`

Reply:
Ohh yes